### DISCUSSION TITLE
Creation of Json schema

### DISCUSSION THREAD

**Discussion by kunalshukla11** [json-schema-org/community#523] [General]
How Can I create json shcema efficiently rather than doing it manual

**Comment by Relequestual** [comment]
Why do you think writing it manually is not the most efficient way?

**Reply by gregsdennis** [reply]
I had a user a few months ago that was needing to create a schema that incorporated a `oneOf`-conditional construct bad on an enumeration with over 50 values.

A lot of times it's just about sheer size.

**Comment by gregsdennis** [comment]
There are numerous generation tools on there implementations page.

https://json-schema.org/implementations
